### PR TITLE
Initial setup.

### PR DESCRIPTION
**Pull request by meharoofnajeeb**
@abhilashsajeev @AmalQB @madhukarmmallia

### CODE REVIEW

File: .jsbeautifyrc [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "max_preserve_newlines": 3,
+    "preserve_newlines": true,
+    "space_before_conditional": true,
+    "space_in_paren": false,
+    "unescape_strings": false,
+    "wrap_line_length": 120,
+    "wrap-attributes-indent-size": 2,
+    "wrap-attributes": "force",
+    "preserve-newlines": true
+  }
```
Comment (abhilashsajeev) [inline .jsbeautifyrc]: End lines should have a trailing space

File: gulp/lint.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+/**
+ * JavaScript Build Lint Task
+ */
+gulp.task('lint:build', function () {
+  return gulp.src(paths.scripts)
+    .pipe($.eslint())
+    .pipe($.eslint.format())
+    .pipe($.eslint.failOnError());
+});
```
Comment (abhilashsajeev) [inline gulp/lint.js]: End lines should have a trailing space

**Comment by abhilashsajeev** [comment]
@meharoofnajeeb After mentioned changes are made :shipit:

**Comment by meharoofnajeeb** [comment]
@abhilashsajeev Mentioned changes have been made.

**Pull request merged**